docs/tutorials/workflow.md

```diff
@@ -36,45 +35,38 @@ print(f"Hamiltonian of type {type(H)} and shape {H.shape}.")
 ```
 
 ```text
-State of type <class 'torch.Tensor'> and shape torch.Size([2, 1]).
-Hamiltonian of type <class 'torch.Tensor'> and shape torch.Size([2, 2]).
+State of type <class 'jaxlib.xla_extension.ArrayImpl'> and shape (2, 1).
+Hamiltonian of type <class 'jaxlib.xla_extension.ArrayImpl'> and shape (2, 2).
 ```
 
-In dynamiqs, all quantum objects are defined directly with PyTorch [Tensors](https://pytorch.org/docs/stable/tensors.html), and without subclassing. This allows for easy interfacing with PyTorch utility functions, and avoids runtime overheads. Also, all quantum objects have at least two dimensions to avoid systematic reshaping or coding mistakes (e.g. trying to multiply a ket and an operator in the wrong order).
-In particular, kets have a shape `(..., N, 1)`.
+In dynamiqs, all quantum objects are defined directly with [JAX arrays](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.array.html), and without subclassing. This allows for easy interfacing with JAX utility functions, and avoids runtime overheads. Also, all quantum objects have at least two dimensions to avoid systematic reshaping or coding mistakes (e.g. trying to multiply a ket and an operator in the wrong order). In particular, kets have a shape `(..., N, 1)`. Note also that objects are single-precision (`float32` or `complex64`) by default.
```

Comment:
```suggestion
In dynamiqs, all quantum objects are defined directly with [JAX arrays](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.array.html), and without subclassing. This allows for easy interfacing with JAX utility functions, and avoids runtime overheads. Also, all quantum objects have at least two dimensions to avoid systematic reshaping or coding mistakes (e.g. trying to multiply a ket and an operator in the wrong order). In particular, kets have a shape `(..., n, 1)`. Note also that objects are single-precision (`float32` or `complex64`) by default.
```